doc/smart-answer-flow-development/creating-a-new-smart-answer.md

```diff
@@ -1,8 +1,38 @@
 # Creating a new Smart Answer
 
-This walks through the basics of creating a new Smart Answer.
+There are two methods for creating a new smart answer:
+  1. automatically - via a rails generator, and
+  2. manually.
 
-## 1. Create a new skeleton flow
+## Using the rails `smart_answer` generator
+
+To automatically generate a skeleton smart answer use the provided rails
+generator...
+
+```bash
+$ rails generate smart_answer example_smart_answer
```

Comment:
I'd suggest just:
```bash
$ rails generate smart_answer my_smart_answer
```
as `example` suggests there is something special about the name.

Reply:
Thanks @reggieb - fair point.  Also makes it consistent with line 28.